manifests/f/fairdataihub/FAIRshare/1.0.0/fairdataihub.FAIRshare.locale.en-US.yaml

```
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: fairdataihub.FAIRshare
+PackageVersion: 1.0.0
+PackageLocale: en-US
+Publisher: fairdataihub

```

```suggestion
Publisher: Fair Data Innovations Hub
```
to match Control Panel.